### PR TITLE
Move kbdpad_mk2 to kbdpad/mk2 and remove clutter

### PR DESCRIPTION
**Pull request by Erovia**
## Description
According to the board's official product page, it's "coming soon", so I believe this is not (yet) a breaking change.
@ai03-2725 can comment on this.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbdpad/mk2/config.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/*
+ * Split Keyboard specific options, make sure you have 'SPLIT_KEYBOARD = yes' in your rules.mk, and define SOFT_SERIAL_PIN.
+ */
+#define SOFT_SERIAL_PIN D0
+
```
Comment (fauxpark) [inline keyboards/kbdfans/kbdpad/mk2/config.h:46]:
```suggestion
```

File: keyboards/kbdfans/kbdpad/mk2/mk2.c [hunk 2]
```diff
@@ -26,7 +26,7 @@ void matrix_init_kb(void) {
 }
 
 void led_set_kb(uint8_t usb_led) {
```
Comment (fauxpark) [inline keyboards/kbdfans/kbdpad/mk2/mk2.c:28]: Can be changed to `led_update_kb()`
Also `matrix_scan_kb()` and `process_record_kb()` can be deleted.

File: keyboards/kbdfans/kbdpad/mk2/mk2.h [hunk 3]
```diff
@@ -27,17 +27,17 @@
  */
 #define LAYOUT_ortho_6x4( \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbdpad/mk2/mk2.h:28]: Also supports `numpad_6x4`

**Comment by ai03-2725** [comment]
The first GB for this product ran nearly a year ago, and is already well into its second or third production run

File: keyboards/kbdfans/kbdpad/mk2/mk2.h (hunk 3 above)
Comment (Erovia) [reply keyboards/kbdfans/kbdpad/mk2/mk2.h:28]: @ai03-2725 could you help with this one?
Which 3 pads are not used with the numpad_6x4 layout on this pcb?

**Comment by tzarc** [comment]
Needs conflict resolution.

File: keyboards/kbdfans/kbdpad/mk2/mk2.h (hunk 3 above)
Comment (peepeetee) [reply keyboards/kbdfans/kbdpad/mk2/mk2.h:28]: image

**Pull request closed without merging**